hlf/chaincode/admission.md

```diff
@@ -0,0 +1,165 @@
+# Hyperledger Admission Api
+
+In the following, the chaincode api for the "Admission"-Chaincode is described.
+This contains all Transactions and Models provided for this domain.
+The Errors returned are defined [here](errors.md#Errors).
+
+## ContractName "UC4.Admission"
+
+## Transactions
+
+### Add Admission
+- ID = addAdmission
+- Send

```

Comment:
We only want to give a single admissionObject here

Reply:
resolved in https://github.com/upb-uc4/api/pull/83/commits/a5dbd05712adf0066bce93947e1d20c03d41e397